src/main/java/org/example/Person.java

```diff
@@ -127,18 +136,55 @@ public void addSibling(Person person){
             this.siblings.add(person);
```

Comment:
In de uitwerkingen hebben ez bij de methode addParents nog een methode toegevoegd, namelijk addChildtoChildren. Daar zou je nog naar kunnen kijken, ter vergelijking/ inspiratie: https://github.com/hogeschoolnovi/backend-java-family-tree-uitwerkingen/blob/main/src/main/java/Person.java

Reply:
Ga ik doen:)